packages/node_modules/@ciscospark/internal-plugin-board/src/realtime.js

```diff
@@ -96,13 +87,20 @@ const RealtimeService = Mercury.extend({
       };
     }
 
-    // use mercury socket if it is shared
-    return this.spark.internal.feature.getFeature(`developer`, `web-shared-mercury`)
-      .then((isSharingMercuryFeatureEnabled) => {
-        if (isSharingMercuryFeatureEnabled && this.isSharingMercury) {
-          return this.spark.internal.mercury.socket.send(data);
-        }
-        return this.socket.send(data);
+    return realtimeChannel.send(data);
```

Comment:
I wonder of realtimeChannel should have a different name. Doesn't "channel" already refer to (approximately) the whiteboard in question?